### ISSUE TITLE
Can I run SPINDLE with OpenMPI launcher without MPIR?

### ISSUE THREAD

**Issue by fukai-t** [hpc/Spindle#35]
I have question about SPINDLE with OpenMPI launcher.

Is there any way to use SPINDLE with OpenMPI launcher without MPIR?
For example, can SPINDLE run with PMIx instead of MPIR?

If there is no way currently, is there any plan to support PMIx?

**Comment by mplegendre** [comment]
There's no PMIx support, and it's not on my near-term road map.

Are you running any resource managers above OpenMPI?  There are other options for SLURM, and upcoming options for LSF.

**Comment by fukai-t** [comment]
Thank you for your answer.
Unfortunately, my environment does not have SLURM nor LSF.

In my situation, is there a possibility to run SPINDLE with `--hostbin` option with a command to return IPs of the MPI nodes? (e.g. `spindle --hostbin=myhostbin mpiexec -np ${num_of_proc} spindlemarker mpiapp.py`)

**Comment by mplegendre** [comment]
Yes, that should work.  Make sure the myhostbin script returns the same nodes that will be used in the job.

Note that the hostbin code probably hasn't been used in several years (that I know of).  So if it has issues, please let me know.

**Comment by fukai-t** [comment]
@mplegendre Thank you for your answer. I'm sorry for my late response. I report my recent status. 
I have tried SPINDLE with hostbin on our cluster. It looks work without MPIR, but I faced another problem (#37) so I have not successed completely run the application with SPINDLE yet.
